cmd/clusterctl/client/cluster/cert_manager_test.go

```diff
@@ -300,7 +300,7 @@ func Test_shouldUpgrade(t *testing.T) {
 							"kind": "Endpoints",
 							"metadata": map[string]interface{}{
 								"annotations": map[string]interface{}{
-									certmanagerVersionAnnotation: "v0.11.0",
+									certmanagerVersionAnnotation: "foo",
```

Comment:
Why did this switch to a non-existent version?

Reply:
Also in general:
Would it make sense to use constants for these versions in tests? It would help in cases like this to keep changes to a single line per test file & reduce the chance of manual mistakes while changing these. (I expect us to change these versions again in the future).

Reply:
> Why did this switch to a non-existent version?

The goal of this test is to test that Endpoints are ignored while determining the version, so it does not really matter what value is used here

> Would it make sense to use constants for these versions in tests?

Generally yes, but being this a backport I'm trying to not deviate from the original commit, so this could be a nice follow-up PR. 
Also, please note that most of the tests are not really linked to the embedded versions, but it is nice to have them up to date.

Reply:
I see that it's already improved on master - so all good :+1: